pioneer:
    updateFreq: 10.00
    teleopGain: -0.01
    obstacleGain: -0.05
lidar:
    gain: 1.00
